data/json/snippets/lab_signs.json

```diff
@@ -0,0 +1,84 @@
+[
+  {
+    "type": "snippet",
+    "category": "lab_access_signage",
+    "//": "signage for the civilian areas (public).",
+    "text": [
+      "***LAB***\nAuthorized Personnel Only",
+      "Restricted\nAccess\nArea",
+      "Be Prepared to present ID",
+      "No civilians beyond this point"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "lab_maintenance_signage",
+    "//": "signage for the maintenance areas (non-public).",
+    "text": [
+      "Restricted\nAccess\nArea",
+      "Be Prepared to present ID",
+      "Keep hands free from machinery",
+      "STOP\nFor Your\nSafety",
+      "NOTICE\nSafety Glasses Required",
+      "DANGER\nKeep Hands Clear\nWhen Equipment\nIs Running"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "lab_security_signage",
+    "//": "signage for the civilian security areas (non-public).",
+    "text": [
+      "Restricted\nAccess\nArea",
+      "Be Prepared to present ID",
+      "Keep guns holstered or locked up",
+      "Tasers Save Lives",
+      "STOP\nWhere's your gun?",
+      "NOTICE\nCheck in with your squad on the hour."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "lab_safety_snippet",
+    "//": "signage for research scientist levels (working lab areas).",
+    "text": [
+      "Keep Equipment Clean and Hygenic",
+      "PPE Required",
+      "<color_yellow_red>Biohazard</color>",
+      "Do not look into laser with remaining eye",
+      "There have been\n[4]\ndays without a life-threatening workplace accident",
+      "Be Alert, Don't get Hurt!",
+      "Safety First\nWork Safely",
+      "<color_white_red>THINK!</color>\n<color_white_red>DON'T CONTAMINATE!</color>\n<color_white>Don't store food in sample refridgerators!</color>",
+      "Report ALL Accidents",
+      "Think!\nDo it the safe way.\nDo it the right way.\nDo it everyday.",
+      "3 causes of accidents are\nI didn't think\nI didn't see\nI didn't know",
+      "Think Safety First!",
+      "Not to be operated by fuckwits",
+      "Stokey the Chemist says:\n'Only YOU can prevent lab fires.  Because you're the asshole who started the last three.'",
+      "<color_yellow_red>Biohazard</color>",
+      "<color_yellow_red>Biohazard</color>",
+      "<color_yellow_red>Biohazard</color>"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "lab_volunteer_snippet",
+    "//": "propaganda signage for the volunteer mutant level.",
+    "text": [
+      "<color_red>For Country</color>\n<color_white>    For</color>\n    <color_cyan>Mom</color>",
+      "<color_white_red>Report any unusual or new symptoms</color>",
```

Comment:
Would they call them symptoms or, say, manifestations?

Reply:
we can do both.  the more variety the better.

Reply:
True. I was thinking that to the subjects, they don't want them to think of illness, so they may not want to call them symptoms. Then again, a _concern_ about symptoms (side effects?) does communicate something positive.